app/views/jobs/dashboard.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1 class="title">Dashboard</h1>
+
+<div class="test">
+  <table class="table">
+    <tr><th>Level of Interest</th><th>Number of Jobs</th></tr>
+    <%= @jobs_by_interest.each do |k, v| %>
+      <tr>
+        <td> <%= k %> </td> <td> <%= v%> </td> </td> 
+      </tr
+    <% end %>
+  </table>
+
+
+  <h2>Top Companies by Interest</h2>
+  <% (@companies_by_interest.length).times do |count| %>
```

Comment:
could we potentially use an ordered list here rather than having to increment a variable while looping?